static/blockchains/categories.json

```diff
@@ -0,0 +1,42 @@
+{
+  "dex": {
+    "name": "DEX",
+    "description": "Decentralized Exchange"
+  },
+  "derivatives": {
+    "name": "Derivatives",
+    "description": "Derivatives"
+  },
+  "nftMarketplace": {
+    "name": "NFT Marketplace",
+    "description": "NFT Marketplace"
+  },
+  "blockchain": {
+    "name": "Blockchain",
+    "description": "Blockchain"
+  },
+  "lending": {
+    "name": "Lending",
+    "description": "Lending"
+  },
+  "staking": {
```

Comment:
Do we add this category on all projects which have their own staking contracts (Anchor, Valkyrie, Astroport, for example), or only for projects whose primary focus of operations/product is staking (such as a liquid staking derivative, or a staking aggregator of sorts, etc.)?
Not sure if we should split this category into two, perhaps, or how we should go about this.